composer.json

```diff
@@ -5,15 +5,17 @@
     "homepage": "https://silverstripe.org",
     "license": "BSD-3-Clause",
     "require": {
+        "php": "^7.3 || ^8.0",
         "silverstripe/recipe-plugin": "^1",
-        "silverstripe/recipe-core": "4.x-dev",
-        "cwp/cwp-core": "2.x-dev",
-        "silverstripe/auditor": "2.x-dev",
-        "silverstripe/environmentcheck": "2.x-dev",
-        "silverstripe/hybridsessions": "2.x-dev"
+        "silverstripe/recipe-core": "^4",
+        "cwp/cwp-core": "^2",
+        "silverstripe/auditor": "^2",
+        "silverstripe/environmentcheck": "^2",
+        "silverstripe/hybridsessions": "^2",
+        "silverstripe/framework": "^4.10"
```

Comment:
The framework requirement is because of phpunit9.  The is far from ideal, since it will prevent this from being using to 4.9 and less. There's an argument to be made to just remove phpunit entirely from this recipe, and thus the framework requirement.

Reply:
I'd probably prefer removing PHPUnit completely and not introducing the framework dependency. If folks want to test projects based on the recipe they can always add/update their project composer.json to add the PHPUnit dev dependency.

Reply:
Can't we just make it add to `require-dev` instead

Reply:
oh yeah :-)

Reply:
Yeah this is fine, composer require will ignore require-dev, so this will be fine if you're using say installer 4.9.0